index.html

```
@@ -60,6 +76,59 @@ <h2 class="thumbnail-title" id="thumbnail-title">
     </div>
+
+    <div class="modal-background">
+      <div class="spinner"></div>
+      <div class="modal">
+        <button class="close-modal" id="closeModal">
+          <img src="./images/modal_button_close.png" />
+        </button>
+        <div class="modal-container">
+          <div class="modal-image">
+            <img
+              src=""
```

HTMLHint (1.9.2)

[error] The attribute [ src ] of the tag [ img ] must have a value.

(src-not-empty)